### ISSUE TITLE
Download and analyze the structure of the .csv (Excel) Training Set and Test Set

### ISSUE THREAD

**Issue by skothare** [skothare/Data-and-Image-processing-using-Machine-Learning-Software-for-Space-Exploration#4]
The Training set for the Iris classification includes four features and the corresponding labels for the respective flower species.

1. Identify the features of Craters and general planetary terrain; for example, crater width and length, depth (identified from the pixels and shade of the craters), general shape of the geographical terrain.

2. Modify the Training set for the flower-classification to fit the crater-classification training set.
